lib/exercises.rb

```diff
@@ -1,19 +1,44 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n)
+# Space Complexity: o(n)
 
 def grouped_anagrams(strings)
```

Comment:
⚠️ As you know this isn't working, but you have the right idea here, just some syntax errors.  

For time complexity, if the strings are limited in size, the time complexity is O(n) and O(n * m log m) where m is the length of the strings (because you're sorting each string).